### PR TITLE
Bug 1274928 - Facebook as an example of hosted app

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=476a941bf18f70a77d01aa5425c79248fa30e509

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8fd609ec52fa32b4588632d63b4053635c633c2a

File: hosted_apps/clock/manifest.webapp [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{
```
Comment (benfrancis) [inline hosted_apps/clock/manifest.webapp]: Is there a reason we can't use a W3C web app manifest here instead?

File: build/config/phone/apps-engineering.list [hunk 2]
```diff
@@ -1,4 +1,5 @@
 apps/*
+hosted_apps/*
```
Comment (benfrancis) [inline build/config/phone/apps-engineering.list]: "web_apps/"? Or should we rename apps/ to chrome/ and call this apps/ ?

File: hosted_apps/clock/metadata.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (benfrancis) [inline hosted_apps/clock/metadata.json]: Why is this needed? Shouldn't we be able to figure out the origin from the start_url? Web apps can't have a separate installOrigin to origin.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ffc68b15ef0fcca0225635c310cf4e631621387

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3ffc68b15ef0fcca0225635c310cf4e631621387
